pipeline.wdl

```diff
@@ -13,28 +15,37 @@ workflow pipeline {
         Array[File] sampleConfigFiles
         String outputDir
         Reference reference
-        IndexedVcfFile dbsnp
+        IndexedVcfFile? dbsnp
         String starIndexDir
         String strandedness
-        File refflatFile
-        File gtfFile
+        File? refflatFile
+        File? referenceGtfFile
+        Array[File] lncRNAdatabases = []
+        Boolean variantCalling = false
+        Boolean lncRNAdetection = false
+        File? cpatLogitModel
+        File? cpatHex
     }
 
     String expressionDir = outputDir + "/expression_measures/"
     String genotypingDir = outputDir + "/multisample_variants/"
 
     # Validation of annotations and dbSNP
-    call biopet.ValidateAnnotation as validateAnnotation {
-        input:
-            refRefflat = refflatFile,
-            gtfFile = gtfFile,
-            reference = reference
-    }
+    # If these are given.
+    # TODO: Discuss: Do we need this?
```

Comment:
This helps detect user error (providing annotations which don't match the reference) early on. I think it should be kept.